### ISSUE TITLE
parsed object: unknown token while reading object - pikepdf inline image parsing error

### ISSUE THREAD

**Issue by G-A-N** [jbarlow83/OCRmyPDF#284]
OCRmyPDF works great except for one file. It is a scientific report available under
`https://core.ac.uk/download/pdf/11034601.pdf`
The error message is:
```
$ ocrmypdf 11034601.pdf  out.pdf
ERROR - parsed object (offset 7): unknown token while reading object; treating as string
```
I' using:
```
$ ocrmypdf --versio
7.0.2
$ qpdf --version
qpdf version 8.1.0
```
Thanks for OCRmyPDF.

**Comment by jbarlow83** [comment]
Fixed for pikepdf 0.3.1/ocrmypdf 7.0.3

**Comment by jbarlow83** [comment]
For issue description see https://github.com/pikepdf/pikepdf/commit/62a37b5d2a72a56ba8b842b522a285426f7d0b3a
